kernel/packages/unity-interface/BrowserInterface.ts

```diff
@@ -339,6 +339,12 @@ export class BrowserInterface {
     setDelightedSurveyEnabled(data.enabled)
   }
 
+  public SetScenesLoadRadius(data: { newRadius: number }) {
```

Comment:
Just in case: we must make sure the `lineOfSightRadius` is properly initialized at the client start according to the saved settings. 

Also, please confirm that we don't have any race condition issues between our settings initialization and the `Lifecycle.initialize` dispatch that comes with `startUnitySceneWorkers()` call on Unity's initialization from kernel side.

Reply:
Another detail: I see that this PR is not addressing how the setting is going to interact with our current LOS url param. The LOS param should be deprecated? Or maybe we have to make sure that if the LOS param exists, the setting is forced at least at the explorer initialization stage.

Reply:
Great points, the saved settings are working correctly in local but not in the build, I'm debugging that issue.

Regarding the LOS URL param, I think that URL param should have priority over the panel setting (unless the setting is changed in runtime after starting the application)

Reply:
Done.
* Now the saved value is used when initializing the explorer
* The LOS URL param is prioritized when initializing the explorer